src/components/userpanel/UserProfile.tsx

```diff
@@ -0,0 +1,629 @@
+import { useEffect, useState, useRef } from "react";
+import styled from "styled-components";
+
+import { useDispatch, useSelector } from "react-redux";
+import { utils, BigNumber } from "ethers";
+import { useCall, useEthers, useLookupAddress } from "@usedapp/core";
+import { Contract } from "@ethersproject/contracts";
+import abiStamps from "../../contracts/aikostamps.json";
+
+import {
+  setShowingNfts,
+  selectGlobalNft,
+  setGlobalAccount,
+  selectHasAikos,
+  setStampsHeld,
+  selectStampsHeld,
+  setMuteAudio,
+  selectMuteAudio,
+} from "../../state/uiSlice";
+
+import check from "../../assets/placeholders/check.png";
+
+import UserNavSocial, { SocialIconType } from "./UserNavSocial";
+
+import JumboStampSystem from "./JumboStampSystem";
+
+import mute from "../../assets/userpanel/mute.png";
+import unmute from "../../assets/userpanel/unmute.png";
+import star from "../../assets/userpanel/stampstar.png";
+import baseaiko from "../../assets/userpanel/aikopfp.gif";
+
+import opensea from "../../assets/svgs/opensea.svg";
+import twitter from "../../assets/svgs/twitter.svg";
+import home from "../../assets/userpanel/homewebsiteicon.png";
+import medium from "../../assets/svgs/medium.svg";
+
+import cursorhover from "../../assets/userpanel/cursorhover.png";
+
+import soundHoverTab from "../../assets/userpanel/Market_SFX_-_TAB_HOVER.wav";
+import soundClickTab from "../../assets/userpanel/Market_SFX_-_TAB_PRESS.wav";
+
+const CONTRACT_ADDR = "0x7f60e977a7b9677be1795efe5ad5516866ab69a6";
+const Interface = new utils.Interface(abiStamps);
+const ContractInstance = new Contract(CONTRACT_ADDR, Interface);
+
+const socialIcons: SocialIconType[] = [
+  {
+    image: twitter,
+    name: "Twitter",
+    link: "https://twitter.com/aikovirtual",
```

Comment:
Maybe we should move social links into some sort of `global.ts` file or `config.ts` or something. As constants, and then import them whenever we need them. Should be a bit cleaner to have all of these links just once and all in one file so that it's easier to update them and maintain.

Reply:
✔️ Done.
Added global variables to global.ts

```javascript
export const openseaLink = "https://opensea.io/collection/aikovirtual";
export const twitterLink = "https://twitter.com/aikovirtual";
export const webLink = "https://aikovirtual.com/";
export const mediumLink = "https://medium.com/@aikovirtual";
export const discordLink = "https://discord.gg/aikovirtual";
```